src/mito2/src/sst/file_purger.rs

```diff
@@ -137,6 +137,13 @@ impl FilePurger for LocalFilePurger {
                 error!(e; "Failed to purge stager with index file, file_id: {}, region: {}",
                     file_meta.file_id(), file_meta.region_id);
             }
+            if let Err(e) = sst_layer
+                .intermediate_manager()
+                .prune_region_dir(&file_meta.region_id)
+                .await
+            {
+                error!(e; "Failed to prune intermediate region directory, region_id: {}", file_meta.region_id);
+            }
```

Comment:
Is it safe to remove the region dir? It may be used by other files of this region.

Reply:
It's only the intermediate files created during indexing. Once the region is purged, these files become useless.